pkg/collector/reconcile/configmap.go

```diff
@@ -38,6 +41,14 @@ func ConfigMaps(ctx context.Context, params Params) error {
 		desiredConfigMap(ctx, params),
 	}
 
+	if params.Instance.Spec.TargetAllocator.Enabled {
+		cm, err := desiredTAConfigMap(params)
+		if err != nil {
+			return fmt.Errorf("failed to parse config: %v", err)
```

Comment:
No need to work on this now, but it's typically better to wrap the error with:

```suggestion
			return fmt.Errorf("failed to parse config: %w", err)
```

(and yes, some of the existing code uses %v as well)